build/edge/run_daemon.sh

```diff
@@ -164,7 +164,7 @@ docker_only_run_edge(){
     edgehubWebsocketUrl=wss://${cloudhub}/e632aba927ea4ac2b575ec1603d56f10/${edgename}/events 
     image=${image:-"kubeedge/edgecore:latest"}
 
-    docker run -d --name edgecore --restart always \
+    docker run -d --name ${containername} --restart always \
```

Comment:
What's the default conatinername?

Reply:
edgecore

Reply:
Please add a default name like we have added for other parameters. edgecore can be default name as it was earlier